pywatts/modules/postprocessing/merger.py

```diff
@@ -0,0 +1,95 @@
+from typing import Dict, Union
+
+import numpy as np
+import xarray as xr
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions import WrongParameterException
+from pywatts.utils._xarray_time_series_utils import numpy_to_xarray
+
+
+class Merger(BaseTransformer):
+    """
+    The merger reduces a two-dimensional time series with multiple values per time step to a uni-variate time series.
+    The first dimension of the input time series is the time-index. The second dimension looks n-1 steps in the past.
+    I.e., the second dimension contains the following values: x(t - n + 1), x(t - n + 2), .. x(t), where t is the
+    current time-index.
```

Comment:
Small revisions on spelling:

The merger reduces a two-dimensional time series with multiple values per time step to a univariate time series.
The first dimension of the input time series is the time index. The second dimension looks n-1 steps into the past 
so it contains the following values: x(t - n + 1), x(t - n + 2), .. x(t), where t is the current time index.